Makefile

```diff
@@ -304,8 +306,11 @@ CPPCHECK        = cppcheck $(CSOURCES) --enable=all --platform=unix64 \
                   $(addprefix -I,$(INCLUDE_DIRS)) \
                   -I/usr/include -I/usr/include/linux
 
-
+ifeq ($(RELEASE),yes)
+TARGET_BASENAME = $(BIN_DIR)/$(FORKNAME)_$(FC_VER)_$(TARGET)
+else
 TARGET_BASENAME = $(BIN_DIR)/$(FORKNAME)_$(FC_VER)_$(TARGET)_$(REVISION)
+endif
```

Comment:
We can also have the revision be the tag such as v4.3.0, in that case we don't have to remove it? This works well with having CI build on tags and create release. Maybe we can consider it later after getting GitHub Actions set up.